.github/workflows/check_links.yml

```diff
@@ -0,0 +1,28 @@
+name: Links
```

Comment:
A better name?

Reply:
What does this workflow do?

Reply:
I believe it tests for broken links. 
cc: @stevhliu can you confirm?

Reply:
Yeah, this uses [lychee](https://lychee.cli.rs/introduction/) to test for broken links